pages/linux/adduser.md

```diff
@@ -0,0 +1,23 @@
+# adduser
+
+> User addition utility.
+
+- Create a new user with a default home directory and prompts the user to set a password:
+
+`adduser {{name}}`
+
+- Create a new user without a home directory and prompts the user to set a password:
+
+`adduser --no-create-home {{name}}`
+
+- Create a new user with a home directory at the specified path and prompts the user to set a password:
```

Comment:
No need to say "and prompts the user to set a password" from here on. Its clear enough that the command will prompt it from the first 2 times.